index.bs

```diff
@@ -176,9 +175,11 @@ Note: This means that an element that is no longer <a>descendant</a> of the {{Do
 
 The {{PerformanceElementTiming/url}} attribute's getter must perform the following steps:
 <div algorithm="PerformanceElementTiming url">
-    1. Let <var>url</var> be <a>request</a>'s <a>resolved URL</a>
-    1. If <var>url</var>'s <a spec=url>scheme</a> is "data:", trim <var>url</var> to its first 100 characters.
-    1. Return <var>url</var>.
+    1. If <a>this</a>'s <a>request</a> is null, return the empty string.
+    1. Let |urlString| be <a>this</a>'s <a>request</a>'s <a>current URL</a>.
```

Comment:
This ends up linking to the wrong definition. The definition in HTML needs to be exported first. That should be a pretty straightforward change. Just add `export` and `for="image request"` attributes.